iocage_lib/iocage.py

```diff
@@ -645,7 +645,14 @@ def create(self,
                     clone=clone,
                     thickconfig=thickconfig
                 ).create_jail()
-        except RuntimeError:
+        except BaseException:
+            if clone:
+                su.run(
```

Comment:
Is this going to be used on CLI and you catch `BaseException` because you want to catch Ctrl-C? If so, some logging can be useful, explaining why there is no immediate reaction to it.

Reply:
`BaseException` is used to catch `SystemExit`, and all the other `Exception` inherited exceptions iocage can throw. `KeyboardInterrupt` is actually handled in `ioc_create.py`